app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -92,3 +92,6 @@ CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
 CONFIG_PROBE=y
 CONFIG_PROBE_DMA_MAX=2
 CONFIG_LOG_TIMESTAMP_64BIT=y
+
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
+CONFIG_COMP_STUBS=y
```

Comment:
?

From `src/audio/Kconfig`
"CONFIG_STUBS: This should only be used in testing environments like fuzzers or CI."

Good example of #9386

cc: @cujomalainey

Reply:
Now being reverted in:
- #9410

Reply:
#9410 results and discussion seem to show a test seems to depend on this!